### DISCUSSION TITLE
Automaticlly dark-mode when I start a new nextjs project

### DISCUSSION THREAD

**Discussion by RevanthGundala** [vercel/next.js#40866] [Help] [answered]
Whenever I create and run a new nextjs program, it automatically is in dark mode for some reason. I was wondering why it's like this and not in light mode to begin with.

<img width="1391" alt="Screen Shot 2022-09-23 at 9 54 26 PM" src="https://user-images.githubusercontent.com/93841932/192080622-e099b0cb-c1d8-45c4-bc2a-a62949d17885.png">

**Comment by gaston-flores** [comment, marked as answer]
Is your system's theme set to dark? The default template used by create-next-app [uses prefers-color-scheme](https://github.com/vercel/next.js/blob/bf0827d0ce42186a101127fc3ce9275d1660ebe9/packages/create-next-app/templates/default/styles/globals.css#L18) to style  the page.

**Reply by RevanthGundala** [reply]
Would there be anyway to remove this besides changing my default system's settings? I tried deleting the globals.css file, but nothing changed.

**Reply by gaston-flores** [reply]
If you delete that block and the one in [Home.module.css](https://github.com/vercel/next.js/blob/bf0827d0ce42186a101127fc3ce9275d1660ebe9/packages/create-next-app/templates/default/styles/Home.module.css#L118) it should look like this

![image](https://user-images.githubusercontent.com/18354222/192112458-2ece8ee3-05d5-4cb8-a0a2-b8e8e569fd1e.png)

But you will probably end up replacing all that css anyway 😛

**Comment by rimzzlabs** [comment]
it's uses prefers-color-scheme. That is normal, because the default template now is trying to give you a nice developer experience when at night 😆

**Reply by RevanthGundala** [reply]
Would there be anyway to remove this besides changing my default system's settings? I tried deleting the globals.css file, but nothing changed.
